From ccfc032fb57e98936ad9dd08ad5e75845b9e74b8 Mon Sep 17 00:00:00 2001
From: Kirill Burtsev <kirill.burtsev@qt.io>
Date: Tue, 2 Jul 2019 17:17:18 +0200
Subject: FIXUP: Disable crash-reports when reporting is disabled

Fix compilation issue because of unknown macro and flag,
as enable_reporting feature is defined for net sub component.

Change-Id: I451dc005fec66ddbf7a5c9de43eccfaafa2fe809
Reviewed-by: Allan Sandfeld Jensen <allan.jensen@qt.io>
---
 chromium/content/public/common/content_features.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chromium/content/public/common/content_features.cc b/chromium/content/public/common/content_features.cc
index e14b7733d9b..ff3f58dc1fc 100644
--- a/src/3rdparty/chromium/content/public/common/content_features.cc
+++ b/src/3rdparty/chromium/content/public/common/content_features.cc
@@ -4,6 +4,7 @@
 
 #include "content/public/common/content_features.h"
 #include "build/build_config.h"
+#include "net/net_buildflags.h"
 
 namespace features {
 
-- 
cgit v1.2.1

